manifests/apache.pp

```diff
@@ -56,18 +55,14 @@
   }
 
   if $deploy {
-
-    include ::apache
-
     certs::keypair { 'apache':
       key_pair   => $apache_cert_name,
       key_file   => $apache_key,
       manage_key => true,
-      key_owner  => $::apache::user,
+      key_owner  => 'root',
+      key_group  => 'root',
       key_mode   => '0440',
-      key_group  => $group,
       cert_file  => $apache_cert,
-      notify     => Service['httpd'],
```

Comment:
Why remove this?

Reply:
I'd rather solve this at another layer, namely puppet-katello and puppet-foreman_proxy_content where we apply this. The benefit is that this class becomes totally self-contained. https://github.com/Katello/puppet-foreman_proxy_content/pull/138 should be merged before though. In particular https://github.com/Katello/puppet-foreman_proxy_content/pull/138/commits/199b334caf617d51ce539e18e7d11f3be8ec2e1b

Reply:
Okay, lets merge the other one first 👍 I'll review it later today.